### ISSUE TITLE
User should be able to browse aepps store

### ISSUE THREAD

**Issue by edwarddikgale** [aeternity/aepp-base#431]

**Comment by j28** [comment]
is this any different from the current version of the browser (which has not been released yet)?

**Comment by edwarddikgale** [comment]
Yes its different the idea is to have some aepp store outside the base aepp possibly as its own independent aepp. Initially it could just have hard coded values but eventually this evolves to a place where ae-dapps are submitted, reviewed and approved like apple's app store
